dev/io.openliberty.jakarta.batch.2.0/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Java Batch 2.0 API
+Bundle-SymbolicName: io.openliberty.jakarta.batch-2.0; singleton:=true
+Bundle-Description: jakarta.batch 2.0 API (Reference Implementation)
+
```

Comment:
Why is the entire set of API source file included in open liberty github instead of just the one or ones that need to be updated for open liberty?  This is an unusual way of building the API bundle.  If most classes are unchanged from the published API jar in maven, we should be able to only update the classes that need updating instead of all of them.  See io.openliberty.jakarta.ejb.4.0 for instance we we just update the one single class that needs to be different than what is published in maven central.  if you need help getting this working let me know.